### PR TITLE
Fixing file header for TEK

### PR DESCRIPTION
**Pull request by christian-kirschnick**

### CODE REVIEW

File: services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/diagnosiskeys/structure/file/TemporaryExposureKeyExportFile.java [hunk 1]
```diff
@@ -47,7 +55,8 @@ private TemporaryExposureKeyExportFile(Collection<TemporaryExposureKey> temporar
    */
   public static TemporaryExposureKeyExportFile fromTemporaryExposureKeys(
       Collection<TemporaryExposureKey> temporaryExposureKeys, String region, long startTimestamp, long endTimestamp) {
-    return new TemporaryExposureKeyExportFile(temporaryExposureKeys, region, startTimestamp, endTimestamp);
+    return new TemporaryExposureKeyExportFile(temporaryExposureKeys, region, startTimestamp,
```
Comment (michael-burwig) [inline services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/diagnosiskeys/structure/file/TemporaryExposureKeyExportFile.java:58]: diff pollution

File: services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/diagnosiskeys/structure/file/TemporaryExposureKeyExportFile.java [hunk 2]
```diff
@@ -17,6 +20,11 @@
  */
 public class TemporaryExposureKeyExportFile extends FileOnDisk {
 
+  /**
+   * The mandatory header for all TEK Export files.
+   */
+  private static final byte[] HEADER = "EK Export v1    ".getBytes(StandardCharsets.UTF_8);
```
Comment (MKusber) [inline services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/diagnosiskeys/structure/file/TemporaryExposureKeyExportFile.java:26]: Are the spaces intended?

Comment (christian-kirschnick) [reply services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/diagnosiskeys/structure/file/TemporaryExposureKeyExportFile.java:26]: Yep, this is according to spec.

File: services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/diagnosiskeys/structure/file/TemporaryExposureKeyExportFile.java (hunk 1 above)
Comment (christian-kirschnick) [reply services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/diagnosiskeys/structure/file/TemporaryExposureKeyExportFile.java]: Sorry for that - auto formatter went nuts

**Pull request merged**